packages/cozy-harvest-lib/src/models/cipherUtils.js

```diff
@@ -79,6 +91,23 @@ export const updateCipher = async (vaultClient, cipherId, data) => {
   }
 }
 
+const isAdditionalField = (fieldName, { identifierProperty }) => {
```

Comment:
Nit: why do we use one named parameter, and one that isn't here?

Reply:
Because to me it is expected that isAdditionalField would run on a field name, and then identifierProperty is part of options.